### PR TITLE
feat(cli-ng): list operations

### PR DESCRIPTION
**Pull request by elsirion**
Allows inspecting the operation log using the CLI client. Builds on #2594.

### CODE REVIEW

File: fedimint-cli/src/client.rs [hunk 1]
```diff
@@ -118,6 +120,10 @@ pub enum ClientCmd {
     },
     /// Print the secret key of the client
     PrintSecret,
+    ListOperations {
```
Comment (dpc) [inline fedimint-cli/src/client.rs:123]: Is this for currently active / all current and past? Would it make sense to add a some filters right away?

File: fedimint-cli/src/client.rs [hunk 2]
```diff
@@ -349,6 +355,51 @@ pub async fn handle_ng_command(
...
+            const ISO8601_CONFIG: iso8601::EncodedConfig = iso8601::Config::DEFAULT
+                .set_formatted_components(iso8601::FormattedComponents::DateTime)
+                .encode();
+            let operations = client
+                .operation_log()
+                .list_operations(limit, None)
+                .await
+                .into_iter()
+                .map(|(k, v)| {
+                    //
```
Comment (dpc) [inline fedimint-cli/src/client.rs:378]: `//` intentional (for formatting maybe?) or leftover?

**Review by dpc — DISMISSED** [review]

File: fedimint-cli/src/client.rs (hunk 1 above)
Comment (elsirion) [reply fedimint-cli/src/client.rs:123]: I'd rather compose queries via `jq`. This is not a user-facing tool.

File: fedimint-cli/src/client.rs (hunk 2 above)
Comment (elsirion) [reply fedimint-cli/src/client.rs]: fixed

**Review by dpc — APPROVED** [review]

**Pull request merged**